src/App.tsx

```diff
@@ -1,24 +1,332 @@
+        setIsThereIssue(false);
+      }, 3000);
+      console.log('Unable to add a todo');
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const updateTempTodo = (value: string) => {
+    setInputValue(value);
+  };
+
+  const handleFormSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsThereIssue(false);
+    setIsLoading(true);
+    setIsPlusOne(true);
+
+    setTempTodo({
+      ...tempTodo,
+      title: inputValue.trim(),
+      id: 0,
+      completed: false,
+      userId: USER_ID,
+    });
+
+    if (inputValue.trim() !== '') {
+      const tempTodoItem: Todo = {
+        title: inputValue.trim(),
+        userId: USER_ID,
+        completed: false,
+        id: getRandomNumber(),
+      };
+
+      try {
+        await client.post('/todos', tempTodoItem);
+        setTodos((prevTodo) => [...prevTodo, tempTodoItem]);
+        setApiResponseReceived(true);
+      } catch (error) {
+        setIsThereIssue(true);
+        setDeleteErrorMessage('Unable to add the todo');
+        timeoutId.current = setTimeout(() => {
+          setIsThereIssue(false);
+        }, 3000);
+        setIsPlusOne(false);
+      } finally {
+        setApiResponseReceived(false);
+        setIsLoading(false);
+        setIsPlusOne(false);
+      }
+    } else {
+      setIsThereIssue(true);
+      setIsPlusOne(false);
+      setIsTitleEmpty('Title can\'t be empty');
+    }
+
+    setInputValue('');
+  };
+
+  const deleteTodo = async (id: number) => {
+    setDeletedTodoId(id);
+
+    try {
+      setIsLoading(true);
+      await client.delete(`/todos/${id}`);
+      setTodos(todos.filter((todo) => todo.id !== id));
+    } catch (error) {
+      setDeleteErrorMessage('Unable to delete the todo');
+      setIsThereIssue(true);
+      timeoutId.current = setTimeout(() => {
+        setIsThereIssue(false);
+      }, 3000);
+      setIsLoading(false);
+      console.log('Unable to delete the todo');
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const updateAllTodo = async () => {
+    setTodoStatusChange(true);
+
+    const everyCompleted = todos.every((item) => item.completed);
+
+    if (everyCompleted) {
+      setIsEveryThingTrue(true);
+    }
+
+    const updatedTodos = todos.map((element) => {
+      if (!everyCompleted && element.completed === false) {
+        setToggleFalseTodosId(prevState => [...prevState, element.id]);
+      }
+
+      return everyCompleted
+        ? { ...element, completed: !element.completed }
+        : { ...element, completed: true };
+    });
+
+    setTodos(updatedTodos);
+
+    try {
+      await Promise.all(
+        updatedTodos.map(async (todoItem) => {
+          await client.patch(`/todos/${todoItem.id}`, {
+            ...todoItem,
+            completed: todoItem.completed,
+          });
+        }),
+      );
+    } catch (error) {
+      console.log('There is an issue updating todos.', error);
```

Comment:
Remove the 'console.log' statement since it is not necessary for production code.